### PR TITLE
Added mono compiling example for Windows

### PR DESCRIPTION
**Pull request by Chaosus**
Just a small example for Windows platform..

### CODE REVIEW

File: development/compiling/compiling_with_mono.rst [hunk 1]
```diff
@@ -48,6 +48,23 @@ Notes
...
+    scons -j6 p=windows tools=yes module_mono_enabled=yes mono_glue=no
+    # Generate glue sources
+    bin\godot.windows.tools.64.mono --generate-mono-glue modules/mono/glue
+
+    ### Build binaries normally
+    # Editor
+    scons -j6 p=windows target=release_debug tools=yes module_mono_enabled=yes
+    # Export templates
+    scons -j6 p=windows target=debug tools=no module_mono_enabled=yes
+    scons -j6 p=windows target=release tools=no module_mono_enabled=yes
```
Comment (neikeq) [inline development/compiling/compiling_with_mono.rst]: I think there is no point in including the -j6 argument in this example.

Comment (Chaosus) [reply development/compiling/compiling_with_mono.rst]: Ok

**Comment by Chaosus** [comment]
@neikeq I've added headers like you said. I did it correct ? Please check, I never works with rst before..

**Comment by neikeq** [comment]
Me neither. Looking at http://docutils.sourceforge.net/docs/user/rst/quickstart.html#sections, I think it should be:

``` rst
Examples
--------

Example (Windows)
~~~~~~~~~~~~~~~~~

Example (x11)
~~~~~~~~~~~~~

```

**Comment by Chaosus** [comment]
Ok, I think im done

**Comment by Chaosus** [comment]
Changed..

**Comment by neikeq** [comment]
Thanks!

**Pull request merged**